src/lib/comms/sol-http-server-impl-microhttpd.c

```diff
@@ -959,6 +1058,81 @@ sol_http_server_send_response(struct sol_http_request *request, struct sol_http_
     return ret;
 }
 
+SOL_API struct sol_http_response_sse *
+sol_http_server_send_response_sse(struct sol_http_request *request,
+    void (*cb)(void *data, const struct sol_http_response_sse *sse),
+    const void *cb_data)
+{
+    int ret;
+    struct sol_http_response_sse *sse;
+    struct MHD_Response *mhd_response;
+
+    SOL_NULL_CHECK(request, NULL);
+    SOL_NULL_CHECK(request->connection, NULL);
+
+    sse = malloc(sizeof(*sse));
```

Comment:
i've added `suspended` after i've written it and forgot to change. I will do it.